common/src/main/java/com/wynntils/models/items/ItemEncodingModel.java

```diff
@@ -50,4 +53,17 @@ public boolean canEncodeItem(WynnItem wynnItem) {
     public Pattern getEncodedDataPattern() {
         return ENCODED_DATA_PATTERN;
     }
+
+    public String makeItemString(WynnItem wynnItem, EncodedByteBuffer encodedItem) {
+        String itemName = "";
+
+        // Gear items are named, but their names are encoded in the data
+        if (shareItemName.get()
```

Comment:
Why is this not mirroring the check below in line 98 in the registry? It seems like we will add the chat name to everything that has a name (isn't that like kind of anything, e.g. `Potion of Healing`?) as long as it is not a gear item; but we will only use the chat name for crafted items. I think we should only append the chat name if it really is needed and used.

Reply:
In fact, I see now that this was how I originally implemented this function...

Reply:
I've done so myself.